duckduckgo/iOS
# Removes iOS phased rollout dry run pixel

Task/Issue URL: https://app.asana.com/0/0/1205515530563288/f
Tech Design URL:
CC:

**Description**:
Removes the incremental feature flag tester and associated pixel & user default key



**Steps to test this PR**:
1. Run the app and make sure no pixel related to the incremental test fires (which shouldn't be possible given that all of that code is now deleted)



**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
